### DISCUSSION TITLE
Runaway cron

### DISCUSSION THREAD

**Discussion by ydnar** [community/community#25178] [Actions]
<p>We have a monorepo with a scheduled workflow to update our Swift Package Manager dependencies. We use Dependabot for non-SPM dependencies, configured with weekly runs early Monday morning.</p>
<p>This morning we discovered GitHub Actions had run our SPM workflow <em>9 times</em>, off master, off the same commit. Prior to this, Dependabot created 10 pull requests for other dependencies.</p>
<p>Why would the scheduler run a master branch scheduled workflow 9 times instead of once?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/c/c85fc2b372efc0d5875e32bd78dc990890aa3fff.png" data-download-href="/uploads/short-url/sAAEVvTmKkDKD2WuHtcNqS0Y7v1.png?dl=1" title="Screenshot 2020-03-23 08.50.58.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/34146/181089258-2924ff7c-4f65-4cd4-bc82-fbc5fa358986.png" alt="Screenshot 2020-03-23 08.50.58.png" data-base62-sha1="sAAEVvTmKkDKD2WuHtcNqS0Y7v1" width="588" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/c/c85fc2b372efc0d5875e32bd78dc990890aa3fff_2_588x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/c/c85fc2b372efc0d5875e32bd78dc990890aa3fff_2_882x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/c/c85fc2b372efc0d5875e32bd78dc990890aa3fff_2_1176x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/c/c85fc2b372efc0d5875e32bd78dc990890aa3fff_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2020-03-23 08.50.58.png</span><span class="informations">1934×1642 290 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/ydnar">@ydnar</a> ,</p>
<p>The schedule events works as expected on my side. Hence, could you please show me the corn setting of your schedule event? And did the pull request will merge&amp;update the schedule setting?</p>
<p>Please change the corn to a new timepoint and observe if multiple workflows still persist. I checked the github <a href="https://www.githubstatus.com/" rel="nofollow noopener">status</a> site, there is a hint on Mar22 that delays for Actions jobs have recovered. I assume it could be a temporary issue.</p>
<p>Thanks</p>

**Comment by ydnar** [comment]
<p>Hi <a class="mention" href="/weide-zhou">@weide-zhou</a>:</p>
<p>Here’s the cron setting for the workflow:</p>
<pre><code>name: Dependencies

on:
  # push:
  # branches:
  # - &lt;your branch name here&gt;
  schedule:
    - cron: "* 13 * * 1" # 13:00 UTC every Monday
</code></pre>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/ydnar">@ydnar</a> ,</p>
<p>In your corn setting, you didn’t specify the first value(minute) but use ‘*’ , which means every minute in 13rd hour  UTC Mon as a trigger. Due to another rule(github minimum interval 5 mins), multiple workflow appears then. </p>
<p>Please specify a value for minute, eg:</p>
<pre><code>schedule:
    - cron: "10 13 * * 1" # 13:10 UTC every Monday
</code></pre>
<p>Thanks.</p>

**Comment by ydnar** [comment]
<p>Thanks!                         </p>
